wagtail/admin/templates/wagtailadmin/tables/references_cell.html

```diff
@@ -3,7 +3,7 @@
         {% for reference in value %}
             <li>
                 {% if edit_url %}
-                    <a href="{{ edit_url }}#content-path-{{ reference.content_path }}">
+                    <a href="{{ edit_url }}#:w:contentpath={{ reference.content_path }}">
```

Comment:
using `contentpath` instead of `content-path` to match `data-contentpath`, but can be convinced otherwise.